### ISSUE TITLE
[BUG] Docs for Ubuntu install refer to group libvirtd but only libvirt exists

### ISSUE THREAD

**Issue by jglick** [code-ready/crc#622]
### General information

  * OS: Linux
  * Hypervisor: libvirt I suppose (docs are very unclear on implications of choosing “native hypervisors” vs. VirtualBox)
  * Did you run `crc setup` before starting it? Yes but not at the time that the problem reported here was observed.

## CRC version
```
version: 1.0.0-beta.5+f2aa58c
OpenShift version: 4.1.14 (embedded in binary)
```
  
## CRC status
```
ERRO Error occurred: Machine "crc" does not exist. Use "crc start" to add a new one. 
```

## CRC config
```
- skip-check-user-in-libvirt-group      : true
```

## Host Operating System
```
NAME="Ubuntu"
VERSION="19.04 (Disco Dingo)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 19.04"
VERSION_ID="19.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=disco
UBUNTU_CODENAME=disco
```

### Steps to reproduce

As per https://code-ready.github.io/crc/:

  1. `sudo apt install qemu-kvm libvirt-daemon libvirt-daemon-system network-manager` ⇒ OK
  2. `sudo usermod -aG libvirtd $(whoami)` ⇒ fails: `usermod: group 'libvirtd' does not exist`
  3. `newgrp libvirtd` ⇒ fails: `newgrp: group 'libvirtd' does not exist`
  4. look in `/etc/group`, see that there is a `libvirt` group, of which I am already a member
  5. `newgrp libvirt` ⇒ OK
  6. `crc config set skip-check-user-in-libvirt-group true` ⇒ `Successfully configured skip-check-user-in-libvirt-group to true`
  7. `crc setup` ⇒ OK
  8. `crc start` ⇒ working so far (as of this writing, up to _Starting OpenShift cluster_)

Am I missing something here? https://github.com/code-ready/crc/pull/277#discussion_r298901438 discusses the two group names.

**Comment by cfergeau** [comment]
I got a bit more details regarding this. I've confirmed that on debian and ubuntu, `libvirt` is the group name which is used. However, for ubuntu:
```
In more recent releases (>= 16.10 Yakkety) the group was renamed to libvirt. Upgraded systems get a new libvirt group with the same gid as the libvirtd group to match that.
```
16.04 is the only supported ubuntu release older than 16.10, but the libvirt version it ships is too old for crc (1.3.1 - https://packages.ubuntu.com/xenial/libvirt-bin ), so we can document we only support 
Ubuntu 18.04 or newer, and drop all mentions of `libvirtd` from the documentation.
Similarly, for debian, oldstable/stretch has a too old libvirt version (3.0.0), but current stable is fine.
